ui/main.qml

```diff
@@ -339,6 +357,15 @@ StatusWindow {
                 }
             }
         }
+        onActiveChanged: {
+            if (!active) {
+                // animation is finished, app main will be shown
+                // open metrics popup only if it has not been seen
+                if(!localAppSettings.metricsPopupSeen) {
```

Comment:
Thanks for this!

One question though: do you think we need to keep this between app launches? 
I think in-memory storage just for this app launch will be enough. If the user doesn't finish the onboarding and closes the app, it makes sense to show this popup again when they restart it IMO.

Reply:
In this concrete situation we need to remember the state, because we show the popup during startup ONLY if it has not been shown earlier.
Eg. when the user updates the application and metrics feature will be introduced, he sees the popup after log in (only once).